Cargo.toml

```
@@ -22,5 +22,5 @@ rust-version = "1.64"
 
 [workspace.dependencies]
 diff = { version = "0.1.12", default_features = false }
-regex = { version = "1", default_features = false }
-regex-syntax = { version = "0.6", default_features = false }
+regex = { version = "1.3", default_features = false }
+regex-syntax = { version = "0.6.12" }
```

```suggestion
regex-syntax = { version = "0.6.12", features=["unicode"] }
```
maybe somehow missed

---

ah, got it. it was default feature.